tests/beta/interfaces/test_qnode_autograd.py

```diff
@@ -48,8 +48,8 @@ def circuit(a):
 
         # without the interface, the tape simply returns an array of results
         assert isinstance(res, np.ndarray)
-        assert res.shape == (1,)
-        assert isinstance(res[0], float)
+        assert res.shape == tuple()

```

Comment:
Since the QNode now returns a scalar by default

Reply:
Wait, is the output behaviour of a qnode changed to current master, or not? Just trying to understand why the test has to change then?